falcon/errors.py

```diff
@@ -2458,6 +2459,63 @@ def description(self, value):
         pass
 
 
+class MediaValidationError(HTTPBadRequest):
+    """400 Bad Request.
+
+    Request media is invalid. This exception is raised by a media validator
+    (such as
+    :func:`jsonschema.validate <falcon.media.validators.jsonschema.validate>`)
+    when ``req.media`` is successfully deserialized, but fails to validate
+    against the configured schema.
+
+    The cause of this exception, if any, is stored in the ``__cause__``
+    attribute using the "raise ... from" form when raising.
+
+    Note:
+        All the arguments should be passed as keyword only. Using them as positional
+        arguments will raise a deprecation warning and will result in an
+        error in a future version of falcon.
```

Comment:
since this is new why we don't just place a `*` in the init?

Reply:
Hm, good point. I've just copypasted from other exceptions :slightly_smiling_face: